<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentsNotAvailableTimesTimeHorizontalForm_template</class>
 <widget class="QDialog" name="StudentsNotAvailableTimesTimeHorizontalForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="The title of the dialog to show/modify the students' not available times constraints in a unified time horizontal overview">Students' not available times time horizontal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="yearsCheckBox">
       <property name="text">
        <string>Years</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="groupsCheckBox">
       <property name="text">
        <string>Groups</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="subgroupsCheckBox">
       <property name="text">
        <string>Subgroups</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="selectedPushButton">
     <property name="text">
      <string comment="It means to toggle on/off the availability of the students in the selected cells of the table">Selected</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="heightTextLabel">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="heightSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QTableWidget" name="naTableWidget"/>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="colorsCheckBox">
     <property name="text">
      <string>Colors</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="filterCheckBox">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QGroupBox" name="filterGroupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="teachersComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="subjectsComboBox"/>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="studentsComboBox"/>
        </item>
        <item>
         <widget class="QCheckBox" name="showRelatedCheckBox">
          <property name="text">
           <string>Show related</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="activityTagsComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StudentsNotAvailableTimesTimeHorizontalForm_template</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StudentsNotAvailableTimesTimeHorizontalForm_template</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
